communitter-frontend/src/utilities/CommentService.js

```diff
@@ -15,11 +15,52 @@ export class CommentService {
             });
 
             if (!commentResponse.ok) {
-                throw new Error("Failed to create comment");
+                throw new Error("Failed to create comment: ", commentResponse);
             }
             return await commentResponse.json(); 
         } catch (error) {
             console.log(error.message);
         }
     }
+
+    static async editComment(comment,commentId){
+        const token = localStorage.getItem(tokenName);
```

Comment:
additionally, if you are going to use this service, you can get the token in the contructor

Reply:
you are right but I kept the same pattern of writing my service. Although I believe it is a badpractice and maybe decreases the readability.